test/unit/uk/gov/hmrc/organisationsdetailsapi/services/CacheServiceSpec.scala

```diff
@@ -54,25 +51,6 @@ class CacheServiceSpec extends AnyWordSpec with Matchers with MockitoSugar with
   }
 
   "cacheService.get" should {
-
-    "return the cached value for a given id and key" in new Setup {
```

Comment:
Is there a reason these tests are deleted? They seem like reasonably valuable tests?